.packit.yaml

```diff
@@ -8,6 +8,14 @@ jobs:
     targets:
     - fedora-rawhide-x86_64
     - fedora-30-x86_64
-    - fedora-29-x86_64
-    - mageia-cauldron-x86_64
-    - opensuse-tumbleweed-x86_64
+    - fedora-31-x86_64
+    - centos-stream-x86_64
+
+- job: test
```

Comment:
`Cannot parse package config. Provided configuration is not valid: 'test' is not one of ['propose_downstream', 'build', 'sync_from_downstream', 'copr_build', 'tests']`

Reply:
aha, tests :)